Gruntfile.js

```diff
@@ -242,8 +241,7 @@ module.exports = (grunt) => {
       outdir: distJS,
       sourcemap: true,
       // Warning: split mode has still a few issues. See https://github.com/okTurtles/group-income/pull/1196
-      splitting: !grunt.option('no-chunks'),
-      watch: false // Not using esbuild's own watch mode since it involves polling.
+      splitting: !grunt.option('no-chunks')
     },
```

Comment:
Here we can see that allowing dangling commas would sometimes result in nicer diffs